Stream-Two/Unit03-Testing/challenges/challenge1/guess_the_number.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+NUMBER_OF_GUESSES = 10
+RANGE_TOP = 10
+RANGE_BOTTOM = 0
+"""
+Uppercase is used to denote these variables are constants
+"""
+
+while True:
+    # generate the random number between 0 and RANGE (RANGE=10)
+    random_number = random.randint(RANGE_BOTTOM, RANGE_TOP)
+
+    """ raw_input in a for loop gives the user a certain amount
+    of guesses (10 in this case)
+    """
+    for i in range(NUMBER_OF_GUESSES):
+        guess = int(raw_input('guess the number: '))
+        if guess == random_number:
+            print 'well done'
+            break
+        elif guess > random_number:
+            print "too high"
+            print "guesses remaining:"
+            print RANGE_TOP - i
```

Comment:
Changed to NUMBER_OF_GUESSES -1 so that the number of remaining guesses reported is correct. (Was one out)